man/yubikey-crypttab.sh

```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: MIT
+
 # Destroy any old key on the Yubikey (careful!)
```

Comment:
This was added by @poettering, changing to CC-0 should be fine.

Reply:
yeah, all fine to be changed to MIT or CC-0 or something like that.